extensions/wikia/CreateNewWiki/SpecialFinishCreate.class.php

```diff
@@ -3,28 +3,40 @@
 class SpecialFinishCreate extends UnlistedSpecialPage {
 
 	public function __construct() {
-		parent::__construct('FinishCreate', 'finishcreate');
+		parent::__construct( 'FinishCreate', 'finishcreate' );
 	}
-	
+
 	public function execute( $par ) {
 		global $wgUser, $wgOut;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)